src/palace.pyx

```diff
@@ -2338,9 +2338,10 @@ cdef class ChorusEffect(BaseEffect):
     """
     cdef alure.EFXCHORUSPROPERTIES properties
 
-    def __init__(self, waveform: int, phase: int,
-                 depth: float, feedback: float, delay: float,
-                 context: Optional[Context] = None) -> None:
+    def __init__(self, waveform: str = 'triangle',
+                 phase: int = 90, depth: float = 0.1,
+                 feedback: float = 0.25, delay: float,
+                 context: Optional[Context] = None) -> None:    
         super().__init__(context)
         self.properties.waveform = waveform
```

Comment:
'Here the input should be int, I think.

Reply:
I think the better way to do it is to set `self.waveform`.